index.bs

```diff
@@ -5259,6 +5314,66 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 ### Events ### {#module-script-events}
 
+#### The script.message Event #### {#event-script-message}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        script.Message = {
+         method: "script.message",
+         params: script.MessageParameters
+       }
+
+       script.MessageParameters = {
+         channel: script.Channel,
```

Comment:
Maybe just the `script.ChannelId`?
```suggestion
         channel: script.ChannelId,
```

Reply:
We don't use the `Id` suffix elsewhere (except `sharedId` / `internalId`, which maybe in retrospect we should have called `object` / `node` or something).

Reply:
`script.Channel` refer to the structure `script.Channel = { type: "channel", value: script.ChannelProperties }`, and I assume we want only the channel name here, which is `script.ChannelId`.

Reply:
Ah, good catch. I fixed this by changing the `Channel` type to `ChannelValue` and `ChannelId` to `Channel`.